src/main.ts

```diff
@@ -1,6 +1,6 @@
 import { bootstrapApplication } from '@angular/platform-browser';
 import { appConfig } from './app/app.config';
 import { AppComponent } from './app/app.component';
-
+import { register } from 'swiper/element-bundle';
```

Comment:
I don't think this element should be registered here. It should be lazy loaded in the module you use it.

As you can see [here](https://github.com/nolimits4web/swiper/blob/869bb843735718c3c77551644cfd717bbf9e264e/src/swiper-element.mjs#L307) it will check if the element is already registered, so it will not throw errors when called multiple times.
It's safe to call this when the lazy module is loaded. Even if multiple modules using this.

Reply:
You can also make a separate module for this to keep the implementation one place and only have this code one place.

```ts
import { NgModule } from '@angular/core';
import { register } from 'swiper/element-bundle';

@NgModule({
  // Define module imports, declarations, etc. here if needed.
})
export class SwiperModule {
  constructor() {
    // This will register the Swiper elements when this module is initialized.
    register();
  }
}
```